tests/integration/test_sns.py

```diff
@@ -46,6 +48,24 @@
 PUBLICATION_RETRIES = 4
 
 
+# copy/pasted from test_sqs.py, utility function
+def queue_exists(sqs_client, queue_url: str) -> bool:
+    """
+    Checks whether a queue with the given queue URL exists.
+
+    :param sqs_client: the botocore client
+    :param queue_url: the queue URL
+    :return: true if the queue exists, false otherwise
+    """
+    try:
+        result = sqs_client.get_queue_url(QueueName=queue_url.split("/")[-1])
+        return result.get("QueueUrl") == queue_url
+    except ClientError as e:
+        if "NonExistentQueue" in e.response["Error"]["Code"]:
+            return False
+        raise
```

Comment:
nit: Instead of copying the function, we usually extract them and migrate them to pytest fixtures. They can then be easily used across all tests by using pytest's fixture injection. You can find lots of examples in `localstack/testing/pytest/fixtures.py`.

Reply:
Thanks a lot, makes sense! It's done!